src/app/util/attribute-metadata.h

```diff
@@ -112,7 +112,7 @@ union EmberAfDefaultOrMinMaxAttributeValue
     /**
      * Actual default value if the attribute size is 2 bytes or less.
```

Comment:
```suggestion
     * Actual default value if the attribute size is 4 bytes or less.
```

Though really, that's not true because of the string case...  Is this all broken (before your changes) if we actually have a one-char string?

Also, the documentation for ptrToDefaultValue needs to be adjusted.

Reply:
From my understand, yes this was broken for 1-character strings